myhoard/controller.py

```diff
@@ -1273,13 +1338,24 @@ def _mark_periodic_backup_requested_if_interval_exceeded(self):
             )
             and (not most_recent_scheduled or time.time() - most_recent_scheduled >= half_backup_interval_s)
         ):
+            incremental_backup_info = None
+            if self._should_schedule_incremental_backup():
+                incremental_backup_info = self.get_incremental_backup_info()
+                if not incremental_backup_info:
+                    self.log.warning(
+                        "Incremental backup is configured but not possible to take, proceeding with full backup"
+                    )
```

Comment:
This should probably emit a metric? Also the decision if we're doing an incremental or full-backup farther up.

Reply:
Probably, I am thinking if this is any useful information. In my mind, failing back to full backup is a normal scenario.
I am wondering how this metric could be used?

Reply:
Well, it would point to an issue. I think the main use would be to annotage all metrics with a tag if it's a full one or an incremental one so we can display that somewhere and can run statistics on that.

Reply:
That's a good idea about the tags and I added that to metrics related to backup/restoration. I still think this metric is a bit redundant, no sure how can I use it and it will be noisy anyway as in the beginning without some metadata only full backups will be take, so the fallback will take place.
Let me know if you are OK with change/answer.

Reply:
I am resolving it for now unless you have a strong opinion :)